cdap-ui/webpack.config.cdap.js

```diff
@@ -58,6 +59,21 @@ if (mode === 'production' || mode === 'build') {
       }
     })
   );
+  //Add Bamboo to environmental variable
+} else if(mode === 'bamboo') {
+  plugins.push(
+    new webpack.DefinePlugin({
+      'process.env':{
```

Comment:
This will be a problem for libraries like React. If we don't use `production` environment then we can't use minified version of the library.